toke.c

```diff
@@ -11634,7 +11634,7 @@ Perl_scan_str(pTHX_ char *start, int keep_bracketed_quoted, int keep_delims, int
         delim_byte_len = 1;
     }
     else {
-        open_delim_code = utf8_to_uvchr_buf((U8*)s, (U8*)PL_bufend,
+        open_delim_code = utf8_to_uv_or_die((U8*)s, (U8*)PL_bufend,
```

Comment:
this one is ok, since neither s nor `PL_bufend` is a const ptr

Reply:
right